### DISCUSSION TITLE
Skywalking is installed, and there is no data after the service is registered

### DISCUSSION THREAD

**Discussion by ThisOldDog** [apache/skywalking#11814] [Q&A]
### Service Image Version

- `elasticsearch:8.12.0`
- `apache/skywalking-ui:9.4.0`：I've tried version `9.7.0` but a lot of menus are missing
- `apache/skywalking-oap-server:9.7.0`

### Java agent

- version: `9.1.0`
- VM Options: `-javaagent:skywalking-agent.jar=agent.service_name=group::service-name,collector.backend_service=opa-host:11800`

### Status quo

- The service is able to register successfully

![image](https://github.com/apache/skywalking/assets/34301142/b8524d41-e531-488d-a08d-6e7ffe3a8ab9)

- However, there is no data except for Instance

![image](https://github.com/apache/skywalking/assets/34301142/21e5b6ad-a54b-448d-9a86-19549c495340)

### Expected

- The container `skywalking-oap` doesn't have any error logs, how can I troubleshoot the problem?

![image](https://github.com/apache/skywalking/assets/34301142/282cabe8-5bde-433f-b248-8cd8babbd668)

**Comment by wu-sheng** [comment]
I don't have magic to tell. The data flow could only be verified by yourself.

One only thing is, the service is oap, not opa. But can't tell more.

**Reply by ThisOldDog** [reply]
Sorry, `opa` was my mistake. The actual configuration uses an **IP**.
Thanks for the reply, I'll go check the configuration information again.

**Reply by wu-sheng** [reply]
Check time and timezone too. If those are not wrong, I can't see the data in the expected latest time range.

**Comment by krishnaprakashthokala** [comment]
@ThisOldDog  Did you install  skywalking on  Kubernets cluster , i trying to install helm charts on cluster , but giving errors.

**Reply by Superskyyy** [reply]
you could try the skywalking-showcase repo first. Then go from there. https://github.com/apache/skywalking-showcase

**Reply by krishnaprakashthokala** [reply]
@Superskyyy  if i try to build showcase , getting below errors.
![image](https://github.com/apache/skywalking/assets/71684984/55388660-f76f-4abc-86e2-dd41614a1291)
Error: rendered manifests contain a resource that already exists. Unable to continue with install: CustomResourceDefinition "certificaterequests.cert-manager.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "demo"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "skywalking-showcase"
helm.go:84: [debug] CustomResourceDefinition "certificaterequests.cert-manager.io" in namespace "" exists and cannot be imported into the current release: invalid ownership metadata; label validation error: missing key "app.kubernetes.io/managed-by": must be set to "Helm"; annotation validation error: missing key "meta.helm.sh/release-name": must be set to "demo"; annotation validation error: missing key "meta.helm.sh/release-namespace": must be set to "skywalking-showcase"
rendered manifests contain a resource that already exists. Unable to continue with install

**Reply by Superskyyy** [reply]
Please use a fresh k8s cluster or delete the certificate manager crd. This is nothing related to skywalking

**Reply by krishnaprakashthokala** [reply]
@Superskyyy  if I have already existing cert manager then how can i do it, because on my system already running other things on kubernetes cluster.

**Reply by Superskyyy** [reply]
**Please run the showcase in a brand new test cluster, otherwise the undeploy process may delete
some resources that you have installed before running this showcase (for example cert-manager).
If you don't do this in a new test cluster, it's all on your own risks!**

Showcase was designed to run on a new cluster, if you already have a system running other things you should be very careful when proceeding. It's quite easy to get another kind or minikube cluster up for such quick experiments. You don't want to risk killing other work when deploying/undeploying the showcase.

**Reply by Superskyyy** [reply]
https://skywalking.apache.org/docs/skywalking-showcase/next/readme/#usage

**Comment by krishnaprakashthokala** [comment]
@Superskyyy i installed skywalking-showcase on new cluster and successfuly deployed it but oap and mongo db pods are getting crashed .
![image](https://github.com/apache/skywalking/assets/71684984/7af0f5d3-27e2-41ef-bb53-dd7bf7a29fd0)
I have enough resources on my vm.

**Comment by krishnaprakashthokala** [comment]
@Superskyyy 
![image](https://github.com/apache/skywalking/assets/71684984/a73212d6-2d59-497e-9172-f30c5b8d77a0)

**Comment by krishnaprakashthokala** [comment]
@Superskyyy 
![image](https://github.com/apache/skywalking/assets/71684984/8215807f-9380-4df7-ad8c-1774eae7ebe0)
